tsconfig.build.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Let's avoid having to maintain 2 separate ts config. 
This will not scale in the future

We should have a base config and, eventually, an override. Let's avoid duplication

Reply:
good idea